src/main/webapp/user-page.html

```diff
@@ -53,5 +57,11 @@ <h1 id="page-title">User Page</h1>
       </form>
     </div>
 
+    <script>
+      function myFunction() {
+        var x = document.getElementById("location_input").value;
```

Comment:
this will generally lead to script injection by the user,

if it's only for testing, consider logging the value to the console instead of putting into HTML directly

Reply:
Great tip :p thanks!

Reply:
Already replaced it with new code